components/raftstore/src/store/fsm/store.rs

```diff
@@ -828,28 +799,69 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport> PollHandler<PeerFsm<EK, ER>, St
     }
 
     fn end(&mut self, peers: &mut [Box<PeerFsm<EK, ER>>]) {
-        self.flush_ticks();
+        let now = TiInstant::now();
+        if self.poll_ctx.trans.need_flush()
+            && now.saturating_duration_since(self.last_flush_msg_time)
+                >= Duration::from_micros(self.poll_ctx.cfg.raft_msg_flush_interval_us)
+        {
+            self.last_flush_msg_time = now;
```

Comment:
In my opinion, timer should count the time between now and the first time a msg is pushed to buffer.

Reply:
It doesn't matter. I use it just to reduce the frequency of `flush`.